### ISSUE TITLE
resume掉精度问题该怎么解决呢？

### ISSUE THREAD

**Issue by diaoa1900** [GuoQuanhao/yolov5-Paddle#10]
您好，我在使用您的代码时发现--resume时精度会下降，并且可能需要训练一些epoch才能恢复到之前的精度，请问该如何做才能做到resume时不掉精度呢（因为AIstudio的GPU每天只有8点算力卡，我所使用的数据集训练一个epoch需要一个小时，只能通过resume来完成整个训练过程）
![image](https://user-images.githubusercontent.com/65444591/191643229-97d327ba-82f0-439a-a659-34c0259244ba.png)

图中的第一个39-45是直接resume的结果

图中的第二个39-48是我认为之前训练时x['learning_rate']和x['momentum']存在，所以尝试在训练开始前给它们赋上warmup结束时的值，但发现效果并未达到预期
`for j, x in enumerate(optimizer._param_groups):`
                    `x['learning_rate'] = np.interp(nw, [0, nw], [hyp['warmup_bias_lr'] if j == 2 else 0.0, scheduler.base_lr * lf(epoch)])`
                    `if 'momentum' in x:`
                       `x['momentum'] = np.interp(nw, [0, nw], [hyp['warmup_momentum'], hyp['momentum']])`

希望能得到您的帮助，谢谢！
